source/module.c

```diff
@@ -322,6 +322,13 @@ int aws_py_raise_error(void) {
     PyErr_Print();
     fprintf(stderr, "Treating Python exception as error %d(%s)\n", aws_error_code, aws_error_name(aws_error_code));
 
+    return aws_last_error();
+}
+
+int aws_py_raise_error(void) {
+
+    int aws_error_code = AWS_ERROR_UNKNOWN;
+    aws_error_code = aws_py_translate_py_error();
```

Comment:
trivial: this would be simpler as just `int aws_error_code = aws_py_translate_py_error();`